src/__tests__/field/captcha/recaptchav2.test.jsx

```diff
@@ -32,10 +33,10 @@ describe('RecaptchaV2', () => {
       document.getElementById('renderTest').remove();
     });
     it('injects the script', () => {
-      const mockLock = createLockMock({ provider: 'recaptchav2', siteKey: 'mySiteKey' });
-      render(mockLock, document.getElementById('renderTest'), {});
-      expect(document.body.innerHTML).toBe(
-        '<div id="renderTest"></div><script src="https://www.google.com/recaptcha/api.js?hl=en-US"></script>'
+      ReCAPTCHA.loadScript({ hl: 'en-US' }, document.body);
```

Comment:
@stevehobbsdev language is the same as the line above, but could this support the language lock is configured to?

Reply:
This is a unit test only of the recaptcha component, when this component is instantiated whitin the "captchapane" the language sent is the lang used to configure lock